src/components/Hero/index.tsx

```diff
@@ -4,18 +4,22 @@ import Button from 'components/Button';
 
 interface Props {
   backgroundSrc: string;
+  children?: ReactNode;
   className?: string;
-  primaryAction: () => void;
-  primaryActionLabel: string;
-  secondaryAction: () => void;
-  secondaryActionLabel: string;
+  hasActions?: boolean;
```

Comment:
Is there any particular case where the hero change?
I mean this prop indicates that we are using the hero for multiple proposes?

Reply:
Yes, until now, the Hero it's being used on both pages

Main page
![image](https://user-images.githubusercontent.com/20747535/92042762-797ac400-ed40-11ea-8f94-20bfc5a1d2af.png)

Union page
![image](https://user-images.githubusercontent.com/20747535/92042779-813a6880-ed40-11ea-830d-bf72ad22f74b.png)

Since they almost the same styles except for the bottom buttons, I chose to use the same `Hero` component. Please let me know what you think and let's address these changes on a new PR